internal/service/advancedcluster/fw_data_source_advanced_cluster.go

```diff
@@ -0,0 +1,429 @@
+package advancedcluster
```

Comment:
This file will be renamed to `data_source_advanced_cluster.go` as per naming convention, currently using `fw` so review is easier